server/configs/webapps/embedded/README.txt

```diff
@@ -0,0 +1,7 @@
+Thank you for downloading LabKey Server. For more information about...
+
+- Installing LabKey Server. See https://www.labkey.org/Documentation/wiki-page.view?name=embeddedConfig
```

Comment:
@labkey-mohara what pages should we target here? Do we plan to retain the `embedded` part of these page names?

Reply:
For this PR, I just want to make sure these files get included. I will do a follow-up PR to update the README and application.properties template